### DISCUSSION TITLE
Add to compare

### DISCUSSION THREAD

**Discussion by faierbol** [WebCrateApp/webcrate#48] [Ideas] [answered]
HI all and congratz for this great and beautiful tool!
I would love to see an "Add to compare" feature.
Why?:
My current scenario, is that i am looking to buy a house/ apartment and i am searching on different apps/ websites for one which is suitable to our needs.
The problem here comes when we find multiple homes on multiple websites and we need to compare them (price, how many rooms it has and so on)

What?:
i would like to have an "add to compare list" button and on that list underneath all items/ links a simple configurable table like the following:

![image](https://user-images.githubusercontent.com/6561107/139656564-f6b11ea1-f963-45dc-a656-dfca4fea230f.png)

Think about product compare feature from a ecommerce website. 
Bellow are 2 apps examples found here on GitHub which is pretty similar to what i would like to see on Webcreate:
https://github.com/jieun-lee/product-compare
https://github.com/Klaudia26/Compare-product

**Comment by BetaHuhn** [comment, marked as answer]
Hey, thanks for the suggestion and for the nice words!

A dedicated compare feature similar to the ones you mentioned won't make it into WebCrate any time soon, maybe ever. This a pretty niche feature and out of the scope of a simple and focused tool like WebCrate.

What's planned though is a simple table view in addition to the current grid view of links. It will display all the links in a crate in a list/table with different columns like the title, description and URL.

I'm also planning on adding tags ([#24](https://github.com/WebCrateApp/webcrate/discussions/24)) to links which you could use as custom "metadata" for links, i.e. the rating you have in your screenshot. Tags are intended to help you organize links inside a crate and you will be able to search and filter after them. They can contain any text you like, so it could also be used to add custom data to a link.

This combined with the table view comes pretty close to what you want to achieve. You would be able to see all the links in a crate and compare their tags/custom data.

No guaranties on when those two features will actually make it into WebCrate. Tags are pretty high on my list though.

Let me know if this would be useful to you!

**Reply by faierbol** [reply]
Thanks a lot for your answer. Yes I believe it would be ok.
